tests/puppeteer/test/smoke/installShop.js

```diff
@@ -0,0 +1,54 @@
+// Importing pages
+const INSTALL_PAGE = require('../../pages/Install/install');
+
+let page;
+let INSTALL;
+
+// Init objects needed
+const init = async () => {
+  page = await global.browser.newPage();
+  INSTALL = await (new INSTALL_PAGE(page));
+};
+
+// Scenario
+global.scenario('Install Prestashop', async () => {
+  test('should open the Install page', async () => {
+    await INSTALL.goTo(global.URL_INSTALL);
+    await INSTALL.checkStepTitle(INSTALL.firstStepPageTitle, INSTALL.firstStepFrTitle);
+  });
+  test('should change language to English and check title', async () => {
+    await INSTALL.setInstallLanguage();
+    await INSTALL.checkStepTitle(INSTALL.firstStepPageTitle, INSTALL.firstStepEnTitle);
+  });
+  test('should click on next and go to step \'License Agreements\'', async () => {
+    await INSTALL.nextStep();
+    await INSTALL.checkStepTitle(INSTALL.secondStepPageTitle, INSTALL.secondStepEnTitle);
+  });
+  test('should agree to terms and conditions and go to step \'System compatibility\'', async () => {
+    await INSTALL.agreeToTermsAndConditions();
+    await INSTALL.nextStep();
+  });
+  test('should click on next and go to step \'shop Information\'', async () => {
+    if (!INSTALL.elementVisible(INSTALL.stepFinishedLeftMenu)) {
```

Comment:
Is it possible to indicate that the scenario went through this step ? Might be a good information to have, if in the future the installer starts bringing this page up for example

Reply:
the step before is called 
> should agree to terms and conditions and go to step 'System compatibility'

if step 'System compatibility' was skiped, test will not check title and will not click on next

what about doing this instead ?

`test('should agree to terms and conditions and go to step \'System compatibility\'', async () => {
    await INSTALL.agreeToTermsAndConditions();
    await INSTALL.nextStep();
 if (!INSTALL.elementVisible(INSTALL.stepFinishedLeftMenu)) {
      await INSTALL.checkStepTitle(INSTALL.thirdStepPageTitle, INSTALL.thirdStepEnTitle);
    }
  });
  test('should click on next and go to step \'shop Information\'', async () => {
    if (!INSTALL.elementVisible(INSTALL.stepFinishedLeftMenu)) {
      await INSTALL.nextStep();
    }
    await INSTALL.checkStepTitle(INSTALL.fourthStepPageTitle, INSTALL.fourthStepEnTitle);
  });`

but its a double check of existing element.

Reply:
I was just thinking about a simple `console.log` but whatever float your boat :-P

Reply:
console.log will not appear in our mocha report.

Reply:
Done thanks.